### ISSUE TITLE
Game crashes approaching my underground base

### ISSUE THREAD

**Issue by Snyper41084** [RavienGaming/Reforged-Eden-2#226]
### Reforged Eden 2 Build Number

RE2 Build 16

### Game mode

Survival

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

The game freezes every time I approach my base.  I dug my base into the side of a mountain, no matter whether approaching it on foot or in my SV it crashes when I'm about 30 meters from the hangar door.  If I approach on foot and stop within 100 meters of the base, I cannot connect to the container controllers inside.  I have 3 backups from the last 24 hours and all of them experience this problem.  I have tried performing other activities (warping, mining, entering/leaving atmosphere) before approaching the base again, same result each time.

I unsubscribed/resubscribed, verified game files, etc. I did not try this in vanilla as I do not have a vanilla game, and to create one with the complexity of the base I've already constructed would take too long to get to the same point.

### Error code from console or log (Optional)

-ERR- 64 Losing data when packing (14915, -2147483648, 3249) (PA)
[Client_240718-123757-11.log](https://github.com/user-attachments/files/16286478/Client_240718-123757-11.log)

### Playfield Name

Akua

**Comment by RavienGaming** [comment]
This is likely a game bug and will need to be reproduced and reported using a vanilla save game.
When reproducing a game bug in a vanilla save, make sure you are using the same mode _(singleplayer/co-op/dedicated multiplayer, survival/creative)_ as what you were using when the issue happened in the scenario.
To see a list of known game bugs or to report a new one please see: https://empyriononline.com/forums/bugs.24/

**Comment by GreyWolf234** [comment]
I am able to reproduce this issue also using a save game though my base is an above ground base.  If I stay away from my base the error doesn't occur.  The issue happens as described above.  Now the game crashes on load as I got too close to the base.  If you want my save game let me know and I can get it to you.  Not sure how though.

I followed all the troubleshooting steps but they did not help.   I am unable to reproduce in Vanilla.  

I am able to go back to an earlier backup from yesterday and approach the base with no error.  Not sure what happened between then and now to cause this.
